### ISSUE TITLE
Drop dependency on @discordjs/rest

### ISSUE THREAD

**Issue by RafDevX** [ist-bot-team/ist-discord-bot#74]
Use [`client.application.commands.set`](https://discord.js.org/#/docs/main/stable/class/ApplicationCommandManager?scrollTo=set)

**Comment by RafDevX** [comment]
Should this be closed, @diogotcorreia?

**Comment by diogotcorreia** [comment]
Nope, we're still using `@discordjs/rest` when registering commands
